lib/modules/datasource/packagist/index.ts

```diff
@@ -185,6 +185,13 @@ export class PackagistDatasource extends Datasource {
       const meta = await this.getRegistryMeta(registryUrl);
 
       if (meta.metadataUrl) {
+        if (
```

Comment:
do we really need metaUrl for this? shouldn't this be above that check?

Reply:
It's less risky decision to place it near the place where it's relevant and won't affect other execution branches

Reply:
but shouldn't we skip all other lookups if this list exists? all other lookups would fail anyways.

i think we need a Composer/ packagist Power user to help here.

Reply:
Okay, I think it's quite safe to make it your way